deps/ccommon/rust/ccommon-derive/src/attrs.rs

```diff
@@ -0,0 +1,192 @@
+// ccommon - a cache common library.
+// Copyright (C) 2019 Twitter, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use proc_macro2::TokenStream;
+use quote::ToTokens;
+use syn::parse::*;
+use syn::parse_quote;
+use syn::punctuated::*;
+use syn::spanned::Spanned;
+use syn::*;
```

Comment:
nit: this negates the need for the use on line 19. No need to address in this PR if it's also this way in ccommon

Reply:
Will fix this in a PR to ccommon and then it can be brought over at a later point in time.